### ISSUE TITLE
'edit' is not recognized as an internal or external command, operable program, or batch file.

### ISSUE THREAD

**Issue by rossmpersonal** [MicrosoftDocs/windowsserverdocs#4256]
'edit' is not recognized as an internal or external command, operable program, or batch file.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a33810a-1372-cc85-10df-3f2aad2ae812
* Version Independent ID: 3440fe6d-61d8-3b9a-8316-4db564a2b3da
* Content: [edit](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/edit)
* Content Source: [WindowsServerDocs/administration/windows-commands/edit.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/windows-commands/edit.md)
* Product: **windows-server**
* Technology: **manage-windows-commands**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

**Comment by illfated** [comment]
https://en.wikipedia.org/wiki/MS-DOS_Editor
> Edit is still included in later versions of Windows such as **Windows XP, Windows Vista 32 bit, Windows 7 32 bit**, and **Windows 8 32 bit**. Being a 16-bit DOS application, it will not directly run on 64-bit Windows versions.

To me, it looks like that page should be retired to a subdirectory below the https://docs.microsoft.com/en-us/previous-versions/windows/ directory.

**Comment by rossmpersonal** [comment]
Is it still available in the latest Windows 10 release 32-bit edition and/or the latest Windows Server release 32-bit edition?

If so, it should be updated to note that it applies to 32-bit versions only and should not be moved to a subdirectory below previous versions.

**Comment by illfated** [comment]
Good question. I have not done any research beyond the superficial web search (1 page), but it should be interesting to see if Edit has made it that far. I will be back later with any results, whatever they may be.

**Comment by illfated** [comment]
https://social.technet.microsoft.com/Forums/en-US/dcb9b5ea-ba0c-4d19-98d0-e197ab348df1/dos-editor-in-windows-10

>> Edit.com is a 16 bit application so it would only be available on 32-bit versions of Windows.
>>
>
> Edit.com is available on Windows 10 32-bit but requires NTVDM be installed in order to run.  Attempting to run edit.com without NTVDM will prompt to have it installed but I personally haven't gone further than the prompt.

![image](https://user-images.githubusercontent.com/38162891/80911595-90c88700-8d37-11ea-983e-1270cebd29d3.png)

**Comment by alexanderankin** [comment]
msft should bring this command back

**Comment by illfated** [comment]
> msft should bring this command back

Fair point. I miss it a bit myself, at least the few times I have to work in a command-line only environment, but that happens less and less these days.

Would be nice if Microsoft could port a couple of Linux editors into core-only OS versions, like VIM or nano.

**Comment by illfated** [comment]
By the way, this ticket does not fall within the scope of this repository, which is to aim at making constructive changes to the MS Docs (docs.microsoft.com) documentation pages.

@rossmpersonal & @alexanderankin : Do you want to keep this ticket open, knowing that the current comments don't provide much in the way of actual improvement work to be done in the documents? Unless there are some good arguments to work on for improving the document referenced above, I am inclined to ask for this ticket to be closed.

**Comment by rossmpersonal** [comment]
As I pointed out, the current documentation is not accurate should be updated to fix this inaccuracy. As a result, this ticket should not be closed. This discussion on this ticket appears to have drifted into a discussion of an implied request for enhancement of product functionality that is off-topic and inappropriate for a documentation ticket. Those comments should be to a separate in a repository that covers product request for enhancements.

**Comment by illfated** [comment]
Fair enough, good point. Feedback Hub is the recommended channel for product functionality / software feature requests.

Very well. Let's keep this ticket as a reminder to archive or remove the page https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/edit , because it has not really been part of the OS since Windows XP or Vista.

**Comment by rossmpersonal** [comment]
Did we ever even thoroughly review exactly what OS architectures and versions it is still in or has been removed from or when exactly it was removed? For example, if it is was included in Windows 10 32-bit (I don't know this.), the page should be updated to specifically list which versions it is in and which versions it is not in.

**Comment by illfated** [comment]
Also fair & good point. True, edit.com is only supported in 32-bits Windows, but requires NTVDM be installed in order to run.

**Comment by rossmpersonal** [comment]
That needs to be explicitly stated in the documentation. Also, what is/was the last version of 32-bits Windows which allows the NTVDM to be installed? This needs to be verified.

**Comment by illfated** [comment]
@eross-msft : Sorry for pinging you like this during the holidays, but it would be good to know if you would be able to contact someone who could help us verify the last version of 32-bits Windows which allows the NTVDM to be installed and `edit.com` to run in that environment.

As I mentioned earlier, the Technet discussion https://social.technet.microsoft.com/Forums/en-US/dcb9b5ea-ba0c-4d19-98d0-e197ab348df1/dos-editor-in-windows-10 from February 2016 contains user statements saying that edit.com is supported in 32-bits Windows 10, but requires NTVDM to be installed in order to run.

**Comment by rossmpersonal** [comment]
No. I am not able to contact someone to do that. That knowledge should be available within Microsoft. Thank you.

**Comment by illfated** [comment]
Oh wow. Where have I kept my head for the last year. I did not even consider the possibility that this could be your secondary account. /facepalm 🤦🏻‍♂️

**Comment by eross-msft** [comment]
> Oh wow. Where have I kept my head for the last year. I did not even consider the possibility that this could be your secondary account. /facepalm 🤦🏻‍♂️

I've been on vacation for the past 2 weeks. I'm not sure if this is what you're saying here, but I want to make sure you know this isn't a secondary account for me. My only GiHub account is eross-msft.

**Comment by illfated** [comment]
Thank you for clearing up that misunderstanding. I was completely confused there, because I found it hard to believe that a Microsoft employee would use a secondary private account, as well as using a very different tone of voice in that account. Never mind. Glad we got that cleared off the table.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
